src/parser/h1_parser.py

```
@@ -143,15 +143,16 @@ def _require_vote_total(cls, field_name: str, value: object) -> None:
-        """人気順は数字のほか '--':発売前取消 '**':発売後取消 空白:登録なし を取る。"""
+        """人気順は数字のほか 全桁'-':発売前取消 全桁'*':発売後取消 空白:登録なし。"""
 
         if not isinstance(value, str):
             raise ValueError("H1 Ninki must be a provider value")
         if value == "":
             return
         text = value.strip()
-        if text in {"--", "**"}:
-            # 公式表記は2文字固定（'--':発売前取消 '**':発売後取消）。
+        if text in {"-" * width, "*" * width}:
+            # 取消マーカーは項目幅ぶんの繰り返しで届く（実データの3桁票数系は
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)